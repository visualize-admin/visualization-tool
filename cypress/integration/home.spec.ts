describe("The Home Page", () => {
  it("should redirect to /de", () => {
    cy.request({ url: "/", followRedirect: false }).should((response) => {
      expect(response.status).to.equal(307);
      expect(response.headers.location).to.equal("/de");
    });
  });

  it("successfully loads", () => {
    cy.visit("/");
  });

  it("html lang should be 'de'", () => {
    cy.visit("/");
    cy.get("html").should("have.attr", "lang", "de");
  });

  it("language switch should work", () => {
    cy.visit("/");

    cy.get('a[hreflang="fr"]').click();

    cy.location("pathname").should("equal", "/fr");

    cy.get("html").should("have.attr", "lang", "fr");
  });
});
