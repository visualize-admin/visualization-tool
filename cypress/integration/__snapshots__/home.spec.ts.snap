exports[`GraphQL API > Matches test snapshot #0`] =
{
  "hello": "world"
};
